nativeauthenticator/nativeauthenticator.py

```diff
@@ -69,6 +69,11 @@ class NativeAuthenticator(Authenticator):
         default_value=False,
         help="Deletes FirstUse Authenticator database after the import"
     )
+    allow_2fa = Bool(
+        True,
```

Comment:
Let's turn this off by default to begin with. We can turn it on later if we wish.